src/main/cli/settings.c

```diff
@@ -1086,6 +1086,16 @@ const clivalue_t valueTable[] = {
     { PARAM_NAME_YAW_DEADBAND,      VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_RC_CONTROLS_CONFIG, offsetof(rcControlsConfig_t, yaw_deadband) },
     { "yaw_control_reversed",       VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RC_CONTROLS_CONFIG, offsetof(rcControlsConfig_t, yaw_control_reversed) },
 
+#ifdef USE_ALTHOLD_MODE
+    { PARAM_NAME_ALTHOLD_P, VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldPidP) },
+    { PARAM_NAME_ALTHOLD_I, VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldPidI) },
+    { PARAM_NAME_ALTHOLD_D, VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldPidD) },
+
+    { PARAM_NAME_ALTHOLD_THROTTLE_MIN,    VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1000, 2000 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldThrottleMin) },
+    { PARAM_NAME_ALTHOLD_THROTTLE_MAX,    VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 1000, 2000 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldThrottleMax) },
+    { PARAM_NAME_ALTHOLD_TARGET_ADJUST_RATE,    VAR_UINT8 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_ALTHOLD_CONFIG, offsetof(altholdConfig_t, altHoldTargetAdjustRate) },
```

Comment:
could we reuse gps_resque throttleMin throttleMax?

Reply:
Yes, we could, but I think that would better be done in a different PR.  I plan to review the GPS Rescue code in the light of how effective this code is.  It will take a lot of testing to do that, so a separate PR makes sense.